tmd/Neuron/Neuron.py

```diff
@@ -1,21 +1,25 @@
 '''
 tmd class : Neuron
 '''
+import copy
+import numpy as np
+from tmd.Soma import Soma
+from tmd.Tree import Tree
+from tmd.utils import tree_type
 
 
 class Neuron(object):
     """
     A Neuron object is a container for Trees
     (basal, apical and axon) and a Soma.
     """
+    # pylint: disable=import-outside-toplevel
     from tmd.Neuron.methods import size
     from tmd.Neuron.methods import get_bounding_box
```

Comment:
Why do these imports need to be scoped inside Neuron?

Reply:
Because they are methods of the class. They need to be called from inside the class (same for other core classes i.e. Tree)

Reply:
kk